src/main/resources/static/js/init.js

```diff
@@ -158,56 +147,72 @@ function download(index) {
     var month = element('month', index).val();
     var programName = element('program-name', index).html();
     var isImam = programName.toLowerCase() === NUTRITION_PROGRAM.toLowerCase();
-    var url = downloadUrl.replace('NAME', programName).replace('YEAR', year).replace('MONTH', month).replace('IS_IMAM', isImam);
+    var isFamily = programName.toLowerCase() === FAMILYPLANNING_PROGRAM.toLowerCase();
+    var url = downloadUrl.replace('NAME', programName).replace('YEAR', year).replace('MONTH', month).replace('IS_IMAM', isImam).replace('IS_FAMILYPLANNING', isFamily);
     downloadCommon(url);
 }
 
+
 function downloadFiscalYearReport(index) {
-	var yearRange = element('fiscal-year', index).val();
-	var years = yearRange.split('-');
-	var startYear = years[0];
-	var startMonth = 4; //Shrawan
-	var endYear = years[1];
-	var endMonth = 3; //Ashadh
-	var programName = element('program-name', index).html();
+    var yearRange = element('fiscal-year', index).val();
+    console.log("yearRange="+yearRange)
+    var years = yearRange.split('-');
+    console.log(years)
+    var startYear = years[0];
+    var startMonth = 4; //Shrawan
+    var endYear = years[1];
+    var endMonth = 3; //Ashadh
```

Comment:
@dipakthapaofficial  why these months numbers are hardcoded?

Reply:
Month Value for fiscal year. Ashadh start of fiscal year and shrawan end of fiscal year